### ISSUE TITLE
Invalid Node Definition: Cannot modify read-only node definition

### ISSUE THREAD

**Issue by tzarski0** [cisco-open/sdwan-lab-deployment-tool#13]
## Description

When CML server is preconfigured with default SD-WAN node definitions, they are set in read-only mode. When setup tasks tries to update them, it fails with:
```
Invalid Node Definition: Cannot modify read-only node definition: cat-sdwan-edge.
```

## Expected Behavior

setup task should be able to modify the read-only node definitions

**Comment by tzarski0** [comment]
0038d540202c77787cc0b89fa3c6b1c15d229664

**Comment by tzarski0** [comment]
Fix coming in 2.0.10
